### PR TITLE
Upgrade to Hibernate ORM 5.4.4

### PR DESCRIPTION
**Pull request by Sanne**

### CODE REVIEW

**Comment by gsmet** [comment]
I'm wondering if we should merge this one just after the release so that we have a bit more hindsight.

Merging this just the day before the release makes me a bit nervous.

WDYT?

**Comment by Sanne** [comment]
I'd merge it, it's awesomely better. Also .. which one is the experimental project ;)

**Review by gsmet — APPROVED** [review]
Waiting for CI.

**Pull request merged**
